packages/components/src/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import type { HTMLAttributes } from "react";
+import React, { forwardRef } from 'react'
+
+import { Icon } from '../..'
+
+export interface CardProps extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
+  /**
+  * Specifies the text to be loaded into the header.
+  */
+  title?: string
```

Comment:
**question:** Shouldn't `title` be required? (If not, I think we should remove the fs-card-title div)

Reply:
It should, yes